src/prospectScraper/logs/Prospects.log

```diff
@@ -4,1205 +4,1871 @@ Big Board Rank: 1
 Player: Trevor Lawrence
 Draft Status: Eligible
 Big Board Rank: 2
```

Comment:
Should we actually be setting up these logs to be saved to Github? Aren't they re-createable by re-running the program?

Reply:
They're theoretically re-creatable if the website doesn't change before the next time it is run.
I wouldn't be completely opposed to the file being in a .gitignore file.